Sources/Sentry/SentryUIViewControllerSanitizer.m

```diff
@@ -1,34 +1,11 @@
 #import "SentryUIViewControllerSanitizer.h"
+#import "SentrySwift.h"
 
 @implementation SentryUIViewControllerSanitizer
```

Comment:
`m`: I think we can ditch this class now. I only calls `SwiftDescriptor getDescription`.